sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/models/Field.java

```diff
@@ -759,4 +770,35 @@ public Field setFields(List<Field> fields) {
         this.fields = fields;
         return this;
     }
+
+    /**
+     * Get the hidden property: A value indicating whether the field will be
+     * returned in a search result. This property must be true for key fields,
+     * and must be null for complex fields. You can hide a field from search
```

Comment:
```suggestion
     * and must be true for complex fields. You can hide a field from search
```

Reply:
This one keep the same as retrievable comments.

> This property must be true for key fields, and it must be null for complex fields.

Reply:
Shouldn't keep the Javadocs the same between `hidden` and `retrievable` since they are inverses of each other. So, if the key field needs to be `retrievable = true` it must be `hidden = false/null` and the same applies for complex fields.

Reply:
Scratch this comment, missed the logic where `hidden == null` means `retrievable == null`.